cmd/validator.go

```diff
@@ -8,11 +8,11 @@ import (
 
 func Validate(ctx *cli.Context) error {
 	if ctx.Args().Len() < 1 {
-		return fmt.Errorf("resource name is missing")
+		return fmt.Errorf("The resource name is missing")
```

Comment:
This `the` is not required. We can remove this.

Reply:
ok i'll revert this change.